modules/machineset-gcp-enabling-gpu-support.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="machineset-gcp-enabling-gpu-support_{context}"]
+= Enabling GPU support for a machine set
+
+Google Cloud Platform (GCP) Compute Engine allows users to add GPUs to VM instances. Workloads that benefit from access to GPU resources can perform better on compute machines with this feature enabled. {product-title} on GCP supports NVIDIA GPU models in the A2 and N1 machine series.
+
+.Supported GPU configurations
+|====
+|Model name |GPU type |Machine types ^[1]^
+
+|NVIDIA A100
+|`nvidia-tesla-a100`
+a|* `a2-highgpu-1g`
+* `a2-highgpu-2g`
+* `a2-highgpu-4g`
+* `a2-highgpu-8g`
+* `a2-megagpu-16g`
+// Of these, only `a2-highgpu-1g` is mentioned in the test plan. Should we assume all of these types work, or not?
+
+|NVIDIA K80
+|`nvidia-tesla-k80`
+.5+a|* `n1-standard-1`
+* `n1-standard-2`
+* `n1-standard-4`
+* `n1-standard-8`
+* `n1-standard-16`
+* `n1-standard-32`
+* `n1-standard-64`
+* `n1-standard-96`
+// Of these, only `n1-standard-1` with `nvidia-tesla-p100` is mentioned in the test plan. Should we assume all of these combinations work, or not?
```

Comment:
Also comment to the compatibility with machine types and GPUs. GPUs have described `memory` and `vCPU` limits on GCP docs. Meaning some GPUs wont work with machines types which resources dont meet these criteria. 

Would be nice to add a comment mentioning this to users a pointing them to those links. https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines and https://cloud.google.com/compute/docs/gpus#nvidia_gpus_for_compute_workloads

Maybe even a link to GPU availability https://cloud.google.com/compute/docs/gpus/gpu-regions-zones#gpu_regions_and_zones. But that might not be necessary.

Reply:
Ok, updated as follows:
> For more information about machine types, including specifications, compatibility, regional availability, and limitations, see the GCP Compute Engine documentation about [N1 machine series](https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines), [A2 machine series](https://cloud.google.com/compute/docs/accelerator-optimized-machines#a2_vms), and [GPU regions and zones availability](https://cloud.google.com/compute/docs/gpus/gpu-regions-zones#gpu_regions_and_zones).